docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,883 @@
+   202006170000      run_post_05                           -            -             -       -         -
+   202006170000      run_post_06                           -            -             -       -         -
+
+   Summary of workflow status:
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+     0 out of 1 cycles completed.
+     Workflow status:  IN PROGRESS
+
+Error messages for each specific task can be found in the task log files located in ``$EXPTDIR/log``. 
+
+If everything goes smoothly, you will eventually get the following workflow status table as follows:
+
+.. code-block:: console
+
+   CYCLE                    TASK                       JOBID        STATE   EXIT STATUS   TRIES  DURATION
+   ======================================================================================================
+   202006170000        make_grid                     8854765    SUCCEEDED             0       1       6.0
+   202006170000        make_orog                     8854809    SUCCEEDED             0       1      27.0
+   202006170000   make_sfc_climo                     8854849    SUCCEEDED             0       1      36.0
+   202006170000    get_extrn_ics                     8854763    SUCCEEDED             0       1      54.0
+   202006170000   get_extrn_lbcs                     8854764    SUCCEEDED             0       1      61.0
+   202006170000         make_ics                     8854914    SUCCEEDED             0       1     119.0
+   202006170000        make_lbcs                     8854913    SUCCEEDED             0       1      98.0
+   202006170000         run_fcst                     8854992    SUCCEEDED             0       1     655.0
+   202006170000      run_post_00                     8855459    SUCCEEDED             0       1       6.0
+   202006170000      run_post_01                     8855460    SUCCEEDED             0       1       6.0
+   202006170000      run_post_02                     8855461    SUCCEEDED             0       1       6.0
+   202006170000      run_post_03                     8855462    SUCCEEDED             0       1       6.0
+   202006170000      run_post_04                     8855463    SUCCEEDED             0       1       6.0
+   202006170000      run_post_05                     8855464    SUCCEEDED             0       1       6.0
+   202006170000      run_post_06                     8855465    SUCCEEDED             0       1       6.0
+
+If all the tasks complete successfully, the workflow status in the log file will indicate “SUCCESS." Otherwise, the workflow status will indicate “FAILURE."
+
+
+Launch the Rocoto Workflow Manually
+---------------------------------------
+
+Load Rocoto
+^^^^^^^^^^^^^^^^
+
+Instead of running the ``./launch_FV3LAM_wflow.sh`` script, users can load Rocoto and any other required modules. This gives the user more control over the process and allows them to view experiment progress more easily. 
+
```

Comment:
The Rocoto module is loaded automatically on Tier 1 platforms when using the App (workflow env file), so it should work out-of-the-box on those platforms.

Reply:
Great! I will delete the Level 1 system commands then.